### PR TITLE
Tests email

### PR DESCRIPTION
**Pull request by gymad**
## Description

PR contains php unit tests for class SugarEmailAddress

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [x] My code follows the code style of this project found [here](https://suitecrm.com/wiki/index.php/Coding_Standards).
- [ ] My change requires a change to the documentation.
- [x] I have read the [**How to Contribute**](https://suitecrm.com/wiki/index.php/Contributing_to_SuiteCRM) guidelines.

### CODE REVIEW

File: tests/unit/include/SugarEmailAddress/SugarEmailAddressTest.php [hunk 1]
```diff
@@ -0,0 +1,1973 @@
+<?php
```
Comment (daniel-samson) [inline tests/unit/include/SugarEmailAddress/SugarEmailAddressTest.php]: We will need to make sure we add this test to the new test framework

File: tests/unit/modules/EmailMan/EmailManTest.php [hunk 2]
```diff
@@ -189,7 +189,7 @@ public function testis_primary_email_address()
 
         //test with a dummy email set
         $bean->email1 = 'test@test.com';
-        $this->assertEquals(false, $emailMan->is_primary_email_address($bean));
+        $this->assertEquals(true, $emailMan->is_primary_email_address($bean));
```
Comment (daniel-samson) [inline tests/unit/modules/EmailMan/EmailManTest.php]: We need to ensure that this is also in the new test framework

File: include/database/MssqlManager.php [hunk 3]
```diff
@@ -874,7 +874,7 @@ private function returnOrderBy($sql, $orig_order_match)
 
             // Bug # 44923 - This breaks the query and does not properly filter isnull
             // as there are other functions such as ltrim and rtrim.
-            /* else if (strncasecmp($psql, 'isnull', 6) != 0)
+            /* elseif (strncasecmp($psql, 'isnull', 6) != 0)
```
Comment (daniel-samson) [inline include/database/MssqlManager.php]: Should really remove instead

File: tests/unit/include/SugarEmailAddress/SugarEmailAddressTest.php (hunk 1 above)
Comment (gymad) [reply tests/unit/include/SugarEmailAddress/SugarEmailAddressTest.php]: checked +1

File: tests/unit/modules/EmailMan/EmailManTest.php (hunk 2 above)
Comment (gymad) [reply tests/unit/modules/EmailMan/EmailManTest.php]: checked +1

**Pull request closed without merging**